mod/caching/set/type_plus_right/metric/metric_answer.rb

```diff
@@ -1,12 +1,10 @@
 # cache # of answers for metric
-include_set Abstract::AnswerTableCachedCount, target_type: :answer
-
-def search_anchor
-  { metric_id: left.id }
-end
+include_set Abstract::CachedCount
```

Comment:
This set is now merged with what used to be +all metric values.  So it's much simpler.